src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -794,6 +794,144 @@ RppStatus rppt_ricap_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             RppPtr_t interDstPtr,
+                             Rpp32f pixelationPercentage,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    if (pixelationPercentage < 0 || pixelationPercentage > 100)
+    {
+        std::cout << "\nEnter a valid pixelation percentage value";
```

Comment:
This is a library, not an executable, so pls remove the print and change to:
```
if (pixelationPercentage < 0 || pixelationPercentage > 100)
    return RPP_ERROR_INVALID_ARGUMENTS;
```

Reply:
Change for host and gpu

Reply:
Resolved with https://github.com/r-abishek/rpp/pull/236/commits/3f9993d8e96c61fecebe01d8152a851e0998610a